GitTfs/GitTfs.cs

```diff
@@ -34,22 +36,34 @@ public class GitTfs
             _gitTfsVersionProvider = gitTfsVersionProvider;
             _runner = runner;
             _globals = globals;
-            _stdout = stdout;
+            _logger = logger;
             _bootstrapper = bootstrapper;
         }
 
         public int Run(IList<string> args)
         {
-            InitializeGlobals();
             _globals.CommandLineRun = "git tfs " + string.Join(" ", args);
             var command = ExtractCommand(args);
-            if(RequiresValidGitRepository(command)) AssertValidGitRepository();
             var unparsedArgs = ParseOptions(command, args);
+            UpdateLoggerOnDebugging();
```

Comment:
I don't remember what is the reason. This development stayed a long time in my repository :(